sdk/storage/azure-storage-blobs/src/blob_responses.cpp

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include "azure/storage/blobs/blob_responses.hpp"
+
+#include "azure/storage/blobs/blob_client.hpp"
+
+namespace Azure { namespace Storage { namespace Blobs { namespace Models {
+
+  std::unique_ptr<Azure::Core::Http::RawResponse> StartCopyBlobResult::PollInternal(
+      Azure::Core::Context& context)
+  {
+    unused(context);
```

Comment:
@vhvb1989 why did we avoid using `unused()` and instead went with `(void)` in core.

Looks like storage's reliable stream does the same. We may want to be consistent here:
https://github.com/Azure/azure-sdk-for-cpp/blob/50a4186d9615d327c4cf20d673cec47f0d7e6802/sdk/storage/azure-storage-common/src/reliable_stream.cpp#L15

Reply:
https://github.com/Azure/azure-sdk-for-cpp/issues/1483